_posts/2018-10-16-newpipe-0.14.2-released.md

```diff
@@ -0,0 +1,58 @@
+
+On top of that, he also made the video detail view responsive. This means, next and recommended videos will be displayed at the right side of your screen if your device is large enough.
+
+<img class="no-flow vertical" src="/img/screenshots/shot_video_detail_tablet_dark.png"/>
+
+[@jludden](https://github.com/jludden) implemented an additional list layout for the downloads. This might help with these videos which titles have the same beginnings.
+<!-- TODO: linear layout download image -->
+<img class="no-flow" src="/img/screenshots/shot_05.png"/>
+
+## Improved subscription handling
+
+This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This is helpful whenever a channel was deleted or renamed by YouTube. So if you have had dead channels before, it is now time to clean them up!
+And for those, who have discovered incorrect subscription counts in the past, this will not happen again - hopefully.
+
+## Further changes
+
+[@ritiek](https://github.com/ritiek) added separate gesture controls to the player. This means, you can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.
+
+[@MaX-Lo](https://github.com/MaX-Lo) made the player remember the last aspect ratio. In case you have a 18:9 device or just like to view video in another than the default aspect ratio, you will definitely enjoy this improvement. 
+
+[@coin3x](https://github.com/coin3x) fixed the video queuing behaviour. When the latest queue already finished playing, enqueuing a new video or audio will start the stream by its own.
+
+[@theScrabi](https://github.com/theScrabi) worked on improving NewPipe's localization handling. For devices where the content country and the actual location differ, the number of views and video descriptions are now displayed in the correct language. 
+He also added a new content setting which allows you to set the content language. 
+
+Speaking of localization, we would like to thank all translators who updated NewPipe and added support for two new languages: (Don't remember which, will insert here once Weblate was merged. Taking a look into the commit summary is less time expensive then scrolling through the history) All together, NewPipe comes with translations for nearly [55 languages](https://hosted.weblate.org/projects/newpipe/#languages)! If you find incomplete or missing translations in your language, feel free to [improve them](https://hosted.weblate.org/engage/NewPipe/).
+
+[@BO41](https://github.com/BO41) cleaned up the code which results in a smaller app size and tiny performance improvements.
+
+[@theScrabi](https://github.com/theScrabi) took a look at another bug in this version: after fixing time parsing for formats like "02.56", people can use NewPipe in Finland now.
```

Comment:
I'd add "... and other countries with strange date formatting". Show the Finnish how weird their language how weird their language is!

Reply:
Hm. I wouldn't do that :D